coqbot.toml

```diff
@@ -0,0 +1,18 @@
+[bot]
+name="coqbot"
+email="coqbot@users.noreply.github.com"
+
+[server]
+domain="coqbot.heroku.com"
+
+[github]
+#webhook_secret = "xxx"
+#api_token = "xxx"
```

Comment:
What about adding a comment explaining why this is commented out?: "In the deployed version, we read these secrets from environment variables."

Reply:
We now provide a detailed and commented example configuration file.